### DISCUSSION TITLE
Making code (in body and in YAML) regular sized?

### DISCUSSION THREAD

**Discussion by ryan-p-randall** [chrisgrieser/shimmering-focus#272] [Q&A] [answered]
First of all, I love this theme. It's really a delight to use—thank you so much for all your work on it!   

Now that I'm using Obsidian enough to delve into Tasks, Dataview, Templater, etc. code, I've noticed that any of this type of code I type—and I'm honestly not sure if inline code like Templater or Tasks is considered a code block in the css—is decidedly smaller than the regular font.  

I've spent a few hours digging through this theme's css and trying to understand Obsidian's variables, then made a few css snippets attempting to override what makes code font smaller. Unfortunately, none of them work.  

Would you consider sharing a css snippet that sets the code font size (for inline code, code blocks, and in the YAML header) to the regular font size, and/or making that a setting available through the Style Settings pane?   

I have no guess as to how much more work adding that as an option to the Style Settings pane would be, but I'd be happy to submit a feature request if it's a feature you'd consider.  

Thanks again all your work on this excellent theme!

**Comment by chrisgrieser** [comment, marked as answer]
Hi, thank you for the kind words!

I have added a style setting to change the code block font size.

btw, next time, please open a feature request and not a discussion for such things :)

**Reply by ryan-p-randall** [reply]
Wow—I definitely didn't expect you to create this so quickly. Thank you so much. It works wonderfully!
